fintech-examples/fintech-ui/src/app/redirect-after-consent-denied/redirect-after-consent-denied.component.ts

```diff
@@ -0,0 +1,37 @@
+import { Component, OnInit } from '@angular/core';
+import { Router } from "@angular/router";
+import { Observable, timer } from "rxjs";
+import { map, take } from "rxjs/operators";
+
+@Component({
+  selector: 'app-redirect-after-consent-denied',
+  templateUrl: './redirect-after-consent-denied.component.html',
+  styleUrls: ['./redirect-after-consent-denied.component.scss']
+})
+export class RedirectAfterConsentDeniedComponent implements OnInit {
+
+  seconds = 5;
+  private countDown$: Observable<number>;
+
+  constructor(private router: Router) {
```

Comment:
As I understand there is a call to the backend to be done to destroy assigned redirect cookie and data related to consent initiation. This is missing here